src/main/java/io/reactivex/internal/observers/CallbackCompletableObserver.java

```diff
@@ -72,7 +65,15 @@ public void onError(Throwable e) {
 
     @Override
     public void onSubscribe(Disposable d) {
-        DisposableHelper.setOnce(this, d);
+        if (DisposableHelper.setOnce(this, d)) {
+            try {
```

Comment:
This part needs coverage testing.

Reply:
I agree. I add test code for onSubscribe.